src/components/ConfirmMenuItem/index.tsx

```diff
@@ -82,7 +82,7 @@ export const MenuItem = forwardRef<MenuItemProps, 'button'>((props, ref) => {
     <StylesProvider value={styles}>
       <StyledMenuItem
         {...menuItemProps}
-        onClick={(e: any) => {
+        onClick={(e: React.MouseEvent<HTMLButtonElement, MouseEvent>) => {
```

Comment:
In my opinion, it is better to remove the custom type and let TypeScript do the type inference.

Reply:
I fixed this, tell me if I understood well